File: app/src/main/java/com/naval/todonotesapp/DetailActivity.java
package com.naval.todonotesapp;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.widget.TextView;

import static com.naval.todonotesapp.AppConstant.DESCRIPTION;
import static com.naval.todonotesapp.AppConstant.TITLE;

public class DetailActivity extends AppCompatActivity {

    TextView detailTextViewTitle,detailTextViewDescription;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_detail);
        bindView();
        Intent intent = getIntent();
        detailTextViewTitle.setText(intent.getStringExtra(TITLE));
        detailTextViewDescription.setText(intent.getStringExtra(DESCRIPTION));
    }

    private void bindView() {
        detailTextViewTitle = findViewById(R.id.detailTextViewTitle);
        detailTextViewDescription = findViewById(R.id.detailTextViewDescription);
    }
}

File: app/src/main/java/com/naval/todonotesapp/DetailActivity.kt
package com.naval.todonotesapp

import android.os.Bundle
import android.util.Log
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import com.naval.todonotesapp.AppConstant.DESCRIPTION
import com.naval.todonotesapp.AppConstant.TAG
import com.naval.todonotesapp.AppConstant.TITLE
import com.naval.todonotesapp.R

class DetailActivity : AppCompatActivity() {

    lateinit var detailTextViewTitle :TextView
    lateinit var detailTextViewDescription : TextView

    override fun  onCreate(savedInstanceState : Bundle?){
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_detail)
        bindView()
        val intent = intent
        Log.d(TAG,intent.getStringExtra(TITLE))
        detailTextViewTitle.text = intent.getStringExtra(TITLE)
        detailTextViewDescription.text = intent.getStringExtra(DESCRIPTION)

    }

    private fun bindView() {

        detailTextViewTitle = findViewById(R.id.detailTextViewTitle)
        detailTextViewDescription = findViewById(R.id.detailTextViewDescription)
    }
}